### ISSUE TITLE
bibtex not working

### ISSUE THREAD

**Issue by andreashandel** [epid8060fall2019/William-Norfolk-Project#1]
got an error message about bibtex file. i think reason is that you have .txt ending, should be .bib.
rest looks good.

**Comment by williamnorfolk** [comment]
Dr. Handel,
Thanks for the feedback! I am still hitting a bit of a wall with the bib file. I was able to get the correct file format into the directory now, but I still receive an error when I knit the bookdown. Displayed as below:

`Error reading bibliography .\../MADA_Project_Refs.bib (line 18, column 1):
unexpected "}"
expecting space or ","
Error running filter C:/PROGRA~1/Pandoc/pandoc-citeproc.exe:
Filter returned error status 1
Error: pandoc document conversion failed with error 83
Execution halted
Warning message:
In file.remove(intermediate_html) :
  cannot remove file 'William_Norfolk_Project_Manuscript.utf8.tmp.html', reason 'No such file or directory'`

The no such file or directory error seems to typically mean the path is incorrect, I have checked the relative path for my bib file and everything seems to be correct. My other thought might be that the bib file format is incorrect? I set it to UTF-8 encoding as the template file is but mine still looks relatively different if you view the raw file. Additionally, I attempted to paste the information from my bibtext file into the template references as a test, and I was able to knit the document but all of my citations appeared as `????` in the document. 

Any advice would be appreciated! Thanks!
